### DISCUSSION TITLE
Issue with Syntax Highlighting in Solidity Test File (Foundry/IDE)**

### DISCUSSION THREAD

**Discussion by chapman91** [Cyfrin/foundry-full-course-cu#2086] [Q&A]
I'm encountering an issue with syntax highlighting in my Foundry project. While Solidity code in other files like .sol contracts appears colored correctly, the code in my test file FundMeTest.t.sol is just displayed as plain text.

Has anyone else faced this issue with syntax highlighting in Foundry test files? Any suggestions on how to configure my editor to properly colorize the code in .t.sol files would be greatly appreciated!

**Comment by EngrPips** [comment]
I haven't experienced this before, and it seems weird. I will suggest you close the `vscode` and then reopen it.
